test/integration/traces_test.go

```diff
@@ -16,29 +16,50 @@ import (
 	grpcclient "github.com/grafana/beyla/test/integration/components/testserver/grpc/client"
 )
 
+func testHTTPTracesNoTraceID(t *testing.T) {
+	testHTTPTracesCommon(t, false)
+}
+
 func testHTTPTraces(t *testing.T) {
-	doHTTPGet(t, instrumentedServiceStdURL+"/create-trace?delay=10ms", 200)
+	testHTTPTracesCommon(t, true)
+}
+
```

Comment:
Can we please add a variation of this test with bogus traceparent?

Reply:
Great idea!  I've added a small set of them, with a few variations of breaking the validation.